addons/fluxcd/template.cue

```diff
@@ -0,0 +1,35 @@
+output: {
+	apiVersion: "core.oam.dev/v1beta1"
+	kind:       "Application"
+	spec: {
+		components: [
+			{
+				type: "k8s-objects"
+				name: "fluxcd-ns"
+				properties: objects: [{
+					apiVersion: "v1"
+					kind:       "Namespace"
+					metadata: name: "flux-system"
+				}]
+			},
+		]
+		policies: [
+			{
+				type: "shared-resource"
+				name: "namespace"
+				properties: rules: [{
+					selector: resourceTypes: ["Namespace"]
+				}]
+			},
+			{
+				type: "topology"
+				name: "deploy-fluxcd-ns"
+				properties: {
+					namespace: "flux-system"
+					// Deploy to all clusters.
+					clusterLabelSelector: {}
```

Comment:
If user set `cluster` args, we should allow user deploy adddon to specified clusters.

Reply:
that's better. accepted